week07/errors_test.go

```diff
@@ -0,0 +1,22 @@
+package week07
+
+import "testing"
+
+func TestErrors(t *testing.T) {
+	tcases := []struct {
+		err error
+		msg string
+	}{
+		{ErrInvalidQuantity(-1), "quantity must be greater than 0, got -1"},
+		{ErrProductNotBuilt("product not built"), "product not built"},
+		{ErrInvalidEmployee(-1), "invalid employee number: -1"},
+		{ErrInvalidEmployeeCount(-1), "invalid employee count: -1"},
+		{ErrManagerStopped{}, "manager is stopped"},
```

Comment:
Always use field names.
```suggestion
		{err: ErrInvalidQuantity(-1), msg: "quantity must be greater than 0, got -1"},
```